qemu.go

```diff
@@ -483,7 +500,40 @@ func (q *qemu) setMemoryResources(podConfig PodConfig) ciaoQemu.Memory {
 		MaxMem: memMax,
 	}
 
-	return memory
+	return memory, nil
+}
+
+func getHostMemorySizeKb(memInfoPath string) (uint64, error) {
```

Comment:
Please move this to `hypervisor.go` since it can be reused across different hypervisor implementations.